app/webpack/src/javascript/sb-admin.js

```diff
@@ -51,8 +51,12 @@ $(document).on('turbolinks:load', function() {
         format: 'yyyy-mm-dd',
         orientation: 'bottom'
     });
-    $('#balance_transaction_transactions_attributes_date').datepicker({
+    $('#transaction_date').datepicker({
         format: 'yyyy-mm-dd',
         orientation: 'bottom'
     });
+    $("#q_transactinable_of_CategoryTransaction_type_category_id_in").chosen();
+    $("#q_transactinable_of_BetweenCategoriesTransaction_type_category_from_id_in").chosen();
+    $("#q_transactinable_of_BetweenCategoriesTransaction_type_category_to_id_in").chosen();
```

Comment:
@marderer let's keep names not in capital style
```
q_transactinable_of_between_categories_transaction_type_category_to_id_in
```

Reply:
@sinpai ransack requires for polymorphic associations use capital style for polymorphic type.
[I looked here](https://github.com/activerecord-hackery/ransack/wiki/Polymorphic-searches)